src/Tribe/RSVP/Early_Access/Early_Access.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Tribe\Tickets\RSVP\Early_Access;
+
+
+/**
+ * Class Early_Access
+ *
+ * Handles Early Access for the new RSVP template.
+ *
+ * @since TBD
+ *
+ * @package Tribe\Tickets\RSVP\Early_Access
+ */
+class Early_Access {
+
+	/**
+	 * Returns whether we are using RSVP Early Access or not.
+	 *
+	 * @since TBD
+	 *
+	 * @return bool Whether it's early access or not.
+	 */
+	public function is_rsvp_early_access() {
+		$is_early_access = defined( 'TRIBE_TICKETS_RSVP_EARLY_ACCESS' ) && tribe_is_truthy( TRIBE_TICKETS_RSVP_EARLY_ACCESS );
+
+		return (bool) apply_filters( 'tribe_tickets_is_rsvp_early_access', $is_early_access );
+	}
+}
```

Comment:
I would rather see this live in a template tag that gets included manually inside of the service provider, following what we did with View V2;

https://github.com/moderntribe/the-events-calendar/blob/master/src/functions/views/provider.php#L14-L35